docs/opt_flags.md

```diff
@@ -0,0 +1,89 @@
+# Flags for 
+## Build buildctl --opt 

```

Comment:
Duplicate of https://github.com/moby/buildkit/blob/master/docs/reference/buildctl.md#dockerfile-specific-options